create_pr.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+TARGET=$1
+
+if [ -z "$TARGET" ]; then
+  echo "target required"
+  exit 1
+fi
+
+./vuln-list-update -vuln-list-dir "$VULN_LIST_DIR" -target "$TARGET"
+
+cd "$VULN_LIST_DIR" || exit 1
+
+if [[ -n $(git status --porcelain) ]]; then
+  # List changed files
+CHANGED_FILES=$(git ls-files . --exclude-standard --others | grep "CVE")
+REPO="$REPOSITORY_OWNER/$VULN_LIST_DIR"
+BASE_BRANCH="main"
+# Loop through changed files and create PRs
+for FILE in $CHANGED_FILES; do
+
+  BRANCH_NAME=$(echo $FILE | tr / -)
+  PR_TITLE="Update $FILE"
+  PR_BODY="This PR updates $FILE"
+
+  # Check if a PR with the same branch name already exists
+  open_pr_count=$(gh pr list --state open --base $BASE_BRANCH --repo $REPO | grep $FILE | wc -l)
+
+  if [ "$open_pr_count" -eq 0 ]; then
+    # Create a new branch and push it
+    
+    git checkout -b $BRANCH_NAME
```

Comment:
nit: just to suppress linter issues

```suggestion
    git checkout -b "$BRANCH_NAME"
```